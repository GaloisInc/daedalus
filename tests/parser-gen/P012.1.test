run
P012.ddl
--gen
--input=inputs/P012.1.inp
