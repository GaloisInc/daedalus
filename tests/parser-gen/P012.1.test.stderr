TODO: TC (TCAnnot {tcAnnot = (SourceRange {sourceFrom = SourcePos {sourceIndex = 72, sourceLine = 5, sourceColumn = 16, sourceFile = "./P012.ddl"}, sourceTo = SourcePos {sourceIndex = 76, sourceLine = 5, sourceColumn = 20, sourceFile = "./P012.ddl"}},Annot {annotStates = [], annotContx = Nothing}), tcAnnotExpr = TCSetAny})
CallStack (from HasCallStack):
  error, called at src/Daedalus/ParserGen/Action.hs:808:10 in daedalus-0.1.0.0-inplace:Daedalus.ParserGen.Action
